internal/jujuclient/modelmanager.go

```diff
@@ -208,6 +208,22 @@ func (c Connection) ControllerModelSummary(ctx context.Context, ms *jujuparams.M
 	return errors.E(op, "controller model not found", errors.CodeNotFound)
 }
 
+// ListModelSummaries retrieves the list of model summaries from the controler
+func (c Connection) ListModelSummaries(ctx context.Context, ms jujuparams.ModelSummariesRequest) (jujuparams.ModelSummaryResults, error) {
+	const op = errors.Op("jujuclient.ControllerModelSummary")
+	args := jujuparams.ModelSummariesRequest{
+		UserTag: c.userTag,
```

Comment:
is this correct? shouldn't we just forward the received request?

Reply:
with the `all` flag set to true, you are asking for all model summaries, so the userTag is ignored.

Reply:
is `all` set to true or to `ms.All` which might not be set to true?

Reply:
from the JIMM pkg is called with the flag set to true. Are you suggesting to ignore `ms.All` and set it to true here?